README.md

```diff
@@ -230,6 +230,17 @@ To deploy your `graphql-yoga` server with [Heroku](https://heroku.com), follow t
 
 ## FAQ
 
+### How does `graphql-yoga` compare to `apollo-server` V2?
+
+GraphQL Yoga inspired the external API of apollo-server V2. The following features are still unique to GraphQL Yoga:- 
+
+- Express middleware (using [graphql-middleware](https://github.com/prismagraphql/graphql-middleware))
```

Comment:
this is confusing, especially if you have never heard of the idea of graphql 'middleware'.  Perhaps remove the word 'express' and explain exactly why  a user may want to use `graphql-middleware`

Reply:
Maybe express the fact that yoga is build on top of express.js and can therefore inherit from a rich set of middlewares.